app/models/task.rb

```diff
@@ -1,3 +1,6 @@
 class Task < ApplicationRecord
   belongs_to :status
+  validates :task_name, presence: true, length: { minimum:5 }
+  validates :git_pr_link, presence: true
```

Comment:
Please add regex validation also. Reference: https://stackoverflow.com/questions/1128168/validation-for-url-domain-using-regex-rails